wagl/brdf.py

```diff
@@ -288,22 +300,26 @@ def segmentize_src_poly(length_scale):
     if not bound_poly.intersects(dst_poly):
         return BrdfTileSummary.empty()
 
-    mask = rasterize([(dst_poly, 1)], fill=0, out_shape=(ds_width, ds_height), transform=dst_geotransform)
-    valid_pixels = np.sum(mask)
-    mask = mask.astype(bool)
-    assert mask.shape == ds.shape[:-2]
+    # inside=1, outside=0
+    roi_mask = rasterize([(dst_poly, 1)], fill=0, out_shape=(ds_width, ds_height), transform=dst_geotransform)
+    roi_mask = roi_mask.astype(bool)
+
+    # both ocean_mask and mask shape should be same
+    assert ocean_mask.shape == roi_mask.shape
```

Comment:
Just wondering if it would be better to raise an Error rather than simply assert?